make: freetype-config: Command not found

I cloned the repo in to a folder on my computer, and it is spitting out a couple different errors related to FreeType
```
make: freetype-config: Command not found
make: freetype-config: Command not found
cc -Wall -g     textlabel.c   -lfontconfig -o textlabel
textlabel.c:4:10: fatal error: ft2build.h: No such file or directory
 #include <ft2build.h>
          ^~~~~~~~~~~~
compilation terminated.
make: *** [<builtin>: textlabel] Error 1
```

I am running `Arch Linux`, along with `freetype2 2.9.1` and `fontconfig 2.13.0` packages updated and installed on my system.

--- a/README.md
+++ b/README.md
@@ -32,6 +32,7 @@ For the `textlabel` tool, the following development packages are required (liste
 
 * libfreetype6-dev
 * libfontconfig1-dev
+* pkgconf
 
 `make` builds the main interface binary, the textlabel tool and line2bitmap.
 

--- a/makefile
+++ b/makefile
@@ -2,8 +2,8 @@
 export PREFIX ?= /usr
 
 CFLAGS ?= -Wall -g
-textlabel: CFLAGS += $(shell freetype-config --cflags)
-textlabel: LDLIBS += $(shell freetype-config --libs) -lfontconfig
+textlabel: CFLAGS += $(shell pkg-config --cflags freetype2)
+textlabel: LDLIBS += $(shell pkg-config --libs freetype2) -lfontconfig
 
 all: pt1230 textlabel line2bitmap
 

